chall03/wkorande.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func convertRGBToHex(rs, gs, bs string) string {
+	r, err := strconv.Atoi(rs)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
I think the convert here is too important to be only a Println. It would have been better to use 
```go
log.Fatalln()
```
to exit in case there is a problem.